docs/src/pages/components/masonry/masonry.md

```diff
@@ -0,0 +1,29 @@
+---
+title: React Masonry component
+components: Masonry, MasonryItem
+githubLabel: 'component: Masonry'
+---
+
+# Masonry
+
+<p class="description">Masonry lays out contents of different sizes as blocks of the same width and variable height with configurable gaps.</p>
+
+Masonry maintains a list of content blocks with a consistent width but variable height. The contents are ordered by row. If a row is already filled with the specified number of columns, the next item starts another row, and it is added to the shortest column.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": true}}
+
+## Masonry
+
+{{"demo": "pages/components/masonry/ImageMasonry.js", "bg": true}}
+
+> `<MasonryItem />` can receive not only `<img />` elements but also `<div />` elements as shown in the following demo:
```

Comment:
The `>` is used for quotes. But these are paragraphs introducing the demo so I don't think we need to use `>` here. The "as shown in the following demo" also sounds bit redundant since all of these paragraphs are part of their specific demo (heading).